### PR TITLE
added new tool called Apertium

### PR DESCRIPTION
**Pull request by nicoleSosa**
I added the tool Apertium to the Tools section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,15 +44,24 @@ This list contains tools, books, articles, blogs, courses and everything related
...
 - [Electronjs i18n](https://www.electronjs.org/apps/i18n-manager) - I18n manager for Windows.
+
 - [OmegaT](https://omegat.org/) - Free translation memory application that works on Windows, macOS and Linux.
+
 - [Argos Translate](https://github.com/argosopentech/argos-translate) - Completely open source offline translation app written in Python and based on OpenNMT. It can also be used as a Python library.
+
 - [LibreTranslate](https://github.com/uav4geo/LibreTranslate) - 100% Self-Hosted web app to translate text, based on Argos Translate.
 
+  
+
```
Comment (mbiesiad) [inline README.md:61]: Please don't add free/blank/empty space. Redundant action here

**Review by mbiesiad — CHANGES_REQUESTED** [review]
Please fix mentioned issues. Added redundant new lines \n

File: README.md (hunk 1 above)
Comment (nicoleSosa) [reply README.md:61]: Sorry, I will fix this issue. Thank you.

Comment (nicoleSosa) [reply README.md]: I have made the requested changes to the pull request. Thank you for the review.

**Comment by mbiesiad** [comment]
Hi @nicoleSosa Thank you so much for your contribution here! ✨ Approved 🎉 Have a great day, best wishes

**Comment by nicoleSosa** [comment]
Thank you so much for approving my contribution and hope you have a great day as well.

**Pull request merged**